go.mod

```diff
@@ -1,39 +1,39 @@
 module github.com/karmada-io/dashboard
 
-go 1.23.0
+go 1.23.8
 
-toolchain go1.24.3
+toolchain go1.24.5
```

Comment:
I think we can remove the `tollchain` line here, Karmada v1.14 didn't test against Gov1.24.